src/components/pages/donations.tsx

```diff
@@ -98,18 +96,26 @@ export function Donations() {
             PayPal
           </h3>
           <p>
-            <Button href="https://www.paypal.me/serlo">
+            <a
+              className="serlo-button serlo-make-interactive-transparent-green -ml-1 font-bold pt-1"
+              href="https://www.paypal.me/serlo"
+            >
               <FontAwesomeIcon icon={faPaypal} size="1x" /> Über PayPal spenden
-            </Button>
+            </a>
           </p>
         </DonationCol>
-        <ContactCol>
+        <div
+          className={clsx(
+            'max-md:pt-16 max-md:mt-11 max-md:border-t-8 max-md:border-brand-150',
+            'md:max-w-xs md:absolute md:right-0 md:top-48 md:pr-4'
```

Comment:
Generally, tailwind is using the mobile first approach, e.g. no variant = mobile, with variant = desktop, so I would suggest this equivalent form:

![grafik](https://user-images.githubusercontent.com/13507950/120662927-d51ddd80-c489-11eb-90ab-a48dcea0047a.png)

Reply:
please search `@media (max-width` in our code base. If you are confident we can (and should ^^) replace all of those: fine. But that patterns sometimes saves a lot of code/complexity.
https://tailwindcss.com/docs/breakpoints#max-width-breakpoints

Reply:
I have actually looked through all cases ^^

In most cases, either there are just a few attributes affected or we are already setting the same attribut for other screen sizes.

The "badest" case is the meta menu, which is behaviour as two completely different components -> looking at the code, it may be better to just split it into two different components and change the display property.